src/PluginUpdate.php

```diff
@@ -46,7 +46,7 @@ public static function update_to_2_0_0() {
 		 * 2. Move feed file to a new location.
 		 */
 		$feed_id          = Pinterest_For_Woocommerce()::get_data( 'local_feed_id' );
-		$default_location = Pinterest_For_Woocommerce()::get_base_country() ?? 'US';
+		$default_location = array( Pinterest_For_Woocommerce()::get_base_country() ?? 'US' );
```

Comment:
We are using `$default location` as an array key, which means we cannot have `$default_location` be an array. Line 54 below will be a problem:

https://github.com/woocommerce/pinterest-for-woocommerce/blob/17d3c336545a41ef4c1760285bd98bf1336b88ed/src/PluginUpdate.php#L54

Reply:
Oh man, sorry for that. I have made the changes and left the self review and testing for today.